tests/contrib/pylons/test_pylons.py

```diff
@@ -65,3 +70,39 @@ def test_pylons():
     assert s.duration <= end - start
     eq_(s.error, 0)
     eq_(s.meta.get(http.STATUS_CODE), '200')
+
+def test_pylons_exceptions():
+    writer = DummyWriter()
+    tracer = Tracer()
+    tracer.writer = writer
+    app = FakeWSGIApp()
+    traced = PylonsTraceMiddleware(app, tracer, service="p")
+
+    # successful request
+    assert not writer.pop()
```

Comment:
don't use plain `assert`, use `eq_` or `ok_` from the `nose.tools` package otherwise the error is not significative (e.g. you will just get `AssertError` in the output, unless you're using `py.test`). Feel free to refactor the rest of the file